pyproject.toml

```diff
@@ -17,11 +17,17 @@ classifiers = [
 python = ">=3.9"
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^8.3.2"
-ruff = "^0.6.3"
-pre-commit = "^3.8.0"
 mypy = "^1.11.2"
+pre-commit = "^3.8.0"
+pytest = "^8.3.2"
 pytest-benchmark = "^4.0.0"
+ruff = "^0.6.9"
+
+[tool.poetry.group.docs.dependencies]
+myst-parser = "*"
+rst-to-myst = "*"
```

Comment:
Is this required? Or was it just for the initial conversion from `.rst` to `.md`?